app/serializers/current_user_serializer.rb

```diff
@@ -320,4 +321,8 @@ def can_view_raw_email
   def use_glimmer_topic_list?
     scope.user.in_any_groups?(SiteSetting.experimental_glimmer_topic_list_groups_map)
   end
+
+  def do_not_disturb_channel_position
+    MessageBus.last_id("/do-not-disturb/#{object.id}")
+  end
```

Comment:
This is needed to avoid race conditions in the system specs, but it's a good improvement for production as well.